### PR TITLE
added tests for triangle count on unweighted graphs and graphs with int64 vertex types 

### PR DESCRIPTION
**Pull request by acostadon**
closes https://github.com/rapidsai/cugraph/issues/2615
closes https://github.com/rapidsai/cugraph/issues/2616

replaces #2626 which was closed

### CODE REVIEW

**Comment by acostadon** [comment]
rerun tests

**Comment by acostadon** [comment]
rerun tests

**Review by BradReesWork — APPROVED** [review]

File: python/cugraph/cugraph/tests/test_triangle_count.py
Comment (rlratzel) [inline python/cugraph/cugraph/tests/test_triangle_count.py:114]: I'm assuming this doesn't need to be plural (ie. there's just one file here)?

**Review by rlratzel — APPROVED** [review]
I had a minor comment which need not hold up approval. Feel free to fix if you like but it can be merged without fixing if we need to merge this right away.

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**
